ModularTegustation/tegu_items/representative/research/lcorp.dm

```diff
@@ -231,3 +231,35 @@ GLOBAL_LIST_EMPTY(lcorp_upgrades)
 	cost = AVERAGE_RESEARCH_PRICE
 	corp = L_CORP_REP
 	required_research = /datum/data/lc13research/salesspeed/lvl7
+
+//LOB points
+/datum/data/lc13research/lob/lvl1
+	research_name = "LOB points grant"
+	research_desc = "HQ is allowing you more LOB points budget in exchange for PE."
+	cost = HIGH_RESEARCH_PRICE
+	corp = L_CORP_REP
+
+/datum/data/lc13research/lob/lvl1/ResearchEffect(obj/structure/representative_console/caller, amount = 5)
```

Comment:
Why are you giving across amount? Does this even work?

You can just hard give 5 points.

Reply:
Modified as discussed in PR-Discussions.